content/docs/user-guide/components/reference/terrain/terrain-macro-material.md

```diff
@@ -60,19 +78,45 @@ This structure is used when sending out information about the macro material set
 
 ## TerrainMacroMaterialRequestBus
 
-The `TerrainMacroMaterialRequestBus` is an internal system bus that is only intended for communication between the terrain renderer and the **Terrain Macro Material** component. Other systems generally do not need to use this EBus since nothing outside the terrain system should need any information from the individual component instances. However, if a use case arises, the following request functions on the `TerrainMacroMaterialRequestBus` EBus interface can be used to query individual **Terrain Macro Material** components.
+The `TerrainMacroMaterialRequestBus` is an internal system bus that is only intended for communication between the terrain rendering and editing systems and the **Terrain Macro Material** component. Other systems generally do not need to use this EBus since nothing outside the terrain system should need any information from the individual component instances. However, if a use case arises, the following request functions on the `TerrainMacroMaterialRequestBus` EBus interface can be used to query individual **Terrain Macro Material** components.
 
 | Request Name | Description | Parameter | Return | Scriptable |
 |-|-|-|-|-|
 | `GetTerrainMacroMaterialData` | Returns the `MacroMaterialData` structure assigned to the **Terrain Macro Material** component. | None | [MacroMaterialData](#macromaterialdata) | Yes |
+| `GetTerrainMacroColorImageSize` | Returns the height/width/depth of the color texture in pixels. | None | RHI::Size | No |
+| `GetMacroColorImagePixelsPerMeter` | Returns the number of color texture pixels per meter in world space. | None | Vector2 | No |
 
 ## TerrainMacroMaterialNotificationBus
 
-The `TerrainMacroMaterialNotificationBus` is also an internal system bus that is only intended for communication between the terrain renderer and the **Terrain Macro Material** component. The `TerrainMacroMaterialNotificationBus` EBus notifies listeners through the following notification functions.
+The TerrainMacroMaterialNotificationBus is an internal EBus used by the terrain rendering and editing systems to monitor changes to Terrain Macro Materials. Other systems generally do not need to use this EBus since nothing outside the terrain system should need any information about individual Terrain Macro Materials. However, if a use case arises, the following notification functions on the TerrainMacroMaterialNotificationBus EBus interface can be used to monitor Terrain Macro Material changes.
```

Comment:
Was the removal of the markdown around TerrainMacroMaterialNotificationBus intentional here?

Reply:
It's intentional, I _think_ it's the right thing to do. From the style guide:
https://www.o3de.org/docs/contributing/to-docs/style-guide/format/

"For the first on-page reference to Gems and components, use bold text. Use unformatted text for subsequent references."

I don't believe any of these are the first reference to Terrain Macro Material on this page.

Reply:
The first reference looks to be on line 45, so I just removed it from where it was bold in the request bus too.

Reply:
Good looking out on formatting! Additionally, for EBus names, you can use inline-code style. Same for the following two notif busses
```suggestion
The `TerrainMacroMaterialNotificationBus` is an internal EBus used by the terrain rendering and editing systems to monitor changes to Terrain Macro Materials. Other systems generally do not need to use this EBus since nothing outside the terrain system should need any information about individual Terrain Macro Materials. However, if a use case arises, the following notification functions on the `TerrainMacroMaterialNotificationBus` EBus interface can be used to monitor Terrain Macro Material changes.
```